ui/src/main/webapp/src/app/components/project-list/no-projects-welcome.component.ts

```diff
@@ -0,0 +1,15 @@
+import {Component, Output, EventEmitter} from "@angular/core";
+
+@Component({
+    selector: 'wu-no-projects-welcome',
+    templateUrl: 'no-projects-welcome.component.html'
```

Comment:
Just a detail - but please use `./` for template/styles paths. I know it currently works fine even without it, but some tools might have issue with it and try to load template from absolute (and wrong) path. 
(I think AOT compiler had some issues with it, so it is better for the future to use `./`).